oml/const.py

```diff
@@ -104,3 +104,16 @@ def get_cache_folder() -> Path:
 # hydra provides ability to set its behaviour for convinient backwards compatibility
 # no matter what the current version is
 HYDRA_BEHAVIOUR = "1.1"
+
+# Audio specific constants
+MOCK_AUDIO_DATASET_PATH = CACHE_PATH / "mock_audio_dataset"
+MOCK_AUDIO_DATASET_URL_GDRIVE = "https://drive.google.com/drive/folders/1aYqXBNnERFRIzxr_6cBVO_v_qz2dU7ta"
+MOCK_AUDIO_DATASET_MD5 = "87ba6367ac1231c6be4ccca5e8ace837"
+MOCK_AUDIO_DATASET_CSV_NAME = "df.csv"
+
+AUDIO_EXTENSIONS = [".mp3", ".wav", ".ogg", ".flac"]
+INPUT_FRAME_OFFSET_KEY = "frame_offset"
```

Comment:
Note sure we need this constant, see comments below

Reply:
done